tests/drift_detection/test_kswin.py

```diff
@@ -0,0 +1,69 @@
+from skmultiflow.data.sea_generator import SEAGenerator
+from skmultiflow.drift_detection import KSWIN
+
+def test_kswin_change_detection():
+    """
+    KSWIN Test
+
+    Content:
+    alpha initialisation test.
+    alpha has range from (0,1)
+
+    pre obtained data initialisation test.
+    data must be list
+
+    KSWIN window size initialisation test.
+    0 < stat_size <  w_size
+
+    KSWIN change detector size initialisation test.
+    At least 1 false positive must arisie due to the sensitive alpha, when testing the standard
+    Sea generator
+    """
```

Comment:
Please use the `pytest` mechanism for handling `exceptions` in tests:

https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions